code/modules/client/preference/preferences.dm

```diff
@@ -138,50 +138,9 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 	//Disabilities
 	var/disabilities= 0			//Stores bits for DISABILITY_FLAGs
 	var/disabilities_cures = 0	//Stores bits for CURE_FLAGs
-	var/list/dname_list = list("Nearsighted", "Colourblind", "Blind", "Deaf", "Mute", "Obese", "Swedish Accent", "Chav Accent", "Lisp", "Dizziness")
-	var/list/dflag_list = list(DISABILITY_FLAG_NEARSIGHTED, DISABILITY_FLAG_COLOURBLIND, DISABILITY_FLAG_BLIND, DISABILITY_FLAG_DEAF, DISABILITY_FLAG_MUTE, DISABILITY_FLAG_FAT, DISABILITY_FLAG_SWEDISH, DISABILITY_FLAG_CHAV, DISABILITY_FLAG_LISP, DISABILITY_FLAG_DIZZY)
-	var/list/dcure_list = list(CURE_FLAG_NEARSIGHTED, CURE_FLAG_COLOURBLIND, CURE_FLAG_BLIND, CURE_FLAG_DEAF, CURE_FLAG_MUTE, CURE_FLAG_FAT, CURE_FLAG_SWEDISH, CURE_FLAG_CHAV, CURE_FLAG_LISP, CURE_FLAG_DIZZY)
-	var/list/dblock_list = list()
-
-
-	var/datum/dna/gene/disability/d_gene
-	character.dna.SetDNAState(GLOB.fatblock, TRUE, DNA_SE, TRUE)
-
-
-GLOB.glassesblock
-
-	if(disabilities & DISABILITY_FLAG_BLIND)
-		character.dna.SetDNAState(GLOB.blindblock, TRUE, DNA_SE, TRUE)
-		character.dna.default_blocks.Add(GLOB.blindblock)
-
-	if(disabilities & DISABILITY_FLAG_DEAF)
-		character.dna.SetDNAState(GLOB.deafblock, TRUE, DNA_SE, TRUE)
-		character.dna.default_blocks.Add(GLOB.deafblock)
-
-	if(disabilities & DISABILITY_FLAG_COLOURBLIND)
-		character.dna.SetDNAState(GLOB.colourblindblock, TRUE, DNA_SE, TRUE)
-		character.dna.default_blocks.Add(GLOB.colourblindblock)
-
-	if(disabilities & DISABILITY_FLAG_MUTE)
-		character.dna.SetDNAState(GLOB.muteblock, TRUE, DNA_SE, TRUE)
-		character.dna.default_blocks.Add(GLOB.muteblock)
-
-	if(disabilities & DISABILITY_FLAG_SWEDISH)
-		character.dna.SetDNAState(GLOB.swedeblock, TRUE, DNA_SE, TRUE)
-		character.dna.default_blocks.Add(GLOB.swedeblock)
-
-	if(disabilities & DISABILITY_FLAG_CHAV)
-		character.dna.SetDNAState(GLOB.chavblock, TRUE, DNA_SE, TRUE)
-		character.dna.default_blocks.Add(GLOB.chavblock)
-
-	if(disabilities & DISABILITY_FLAG_LISP)
-		character.dna.SetDNAState(GLOB.lispblock, TRUE, DNA_SE, TRUE)
-		character.dna.default_blocks.Add(GLOB.lispblock)
-
-	if(disabilities & DISABILITY_FLAG_DIZZY)
-		character.dna.SetDNAState(GLOB.dizzyblock, TRUE, DNA_SE, TRUE)
-		character.dna.default_blocks.Add(GLOB.dizzyblock)
-
+	var/list/dname_list = list("Nearsightedness", "Colourblindness", "Blindness", "Deafness", "Mute", "Obesity", "Swedish Accent", "Chav Accent", "Lisp", "Clumsiness")
+	var/list/dflag_list = list(DISABILITY_FLAG_NEARSIGHTED, DISABILITY_FLAG_COLOURBLIND, DISABILITY_FLAG_BLIND, DISABILITY_FLAG_DEAF, DISABILITY_FLAG_MUTE, DISABILITY_FLAG_FAT, DISABILITY_FLAG_SWEDISH, DISABILITY_FLAG_CHAV, DISABILITY_FLAG_LISP, DISABILITY_FLAG_CLUMSY)
+	var/list/dcure_list = list(CURE_FLAG_NEARSIGHTED, CURE_FLAG_COLOURBLIND, CURE_FLAG_BLIND, CURE_FLAG_DEAF, CURE_FLAG_MUTE, CURE_FLAG_FAT, CURE_FLAG_SWEDISH, CURE_FLAG_CHAV, CURE_FLAG_LISP, CURE_FLAG_CLUMSY)
 
```

Comment:
I wanted only one list (dname_list) and then the other lists to populate in a for loop, but the datum level I needed to pull from does not get initialized before I need it ready to be pulled. :(